### PR TITLE
Fixes #6858 - escape HTML tags when update a parameter value in settings

### PR DESCRIPTION
**Pull request by amirfefer**

### CODE REVIEW

File: app/assets/javascripts/settings.js [hunk 1]
```diff
@@ -71,6 +71,8 @@ $(document).ready(function() {
 function onEnterEdit(item){
   item.removeClass("editable");
   item.tooltip('destroy');
+  var unescaped_value = _.unescape(item.find('input').val());
```
Comment (amirfefer) [inline app/assets/javascripts/settings.js:74]: When entering edit mode, the value parameter becomes input tag, hence unescape is needed.

File: app/assets/javascripts/settings.js [hunk 2]
```diff
@@ -26,7 +26,7 @@ $(document).ready(function() {
       else
         editable_value = String(editable_value);
 
-      $(this).html(editable_value);
+      $(this).html((_.escape(editable_value)));
```
Comment (tbrisker) [inline app/assets/javascripts/settings.js]: nitpick: extra parentheses

**Comment by tbrisker** [comment]
Good to merge once nitpick is fixed.

**Comment by amirfefer** [comment]
[test]

**Comment by tbrisker** [comment]
Merged as e108822a1a3ab567ea17d733754ccc9c9447dc8a, thanks @amirfefer!

**Pull request closed without merging**